src/Okta/README.md

```diff
@@ -84,6 +85,21 @@ public function logout(\Illuminate\Http\Request $request)
     return redirect($logoutUrl);
 }
 ```
+#### Refresh Token
+Using a refresh token allows an active user to maintain their session
```

Comment:
Might be worth adding a docs link here?

Reply:
@atymic clarified usage and added link to relevant docs.